packages/faceted-search/src/components/Badges/BadgeOverlay/BadgeOverlay.scss

```diff
@@ -0,0 +1,26 @@
+.tc-badge-link-plus {
+	color: $lochmara;
+	font-size: $font-size-base;
```

Comment:
font size 14px seems to cascade with btn-link, not sure if you need it here.

Reply:
You're right, just remove it 👍